python/src/equistore/labels.py

```diff
@@ -251,6 +251,41 @@ def asarray(self):
     def __contains__(self, label):
         return self.position(label) is not None
 
+    @staticmethod
+    def arange(*args, name="name"):
+        """
+        Creates a very simple `Labels` object.
+
+        The resulting `Labels` only contain one label, whose name
+        is `name`, unless otherwise specified. The values of the
+        labels depend on a set of integer arguments, whose
+        meaning is the same as those of the `numpy.arange` function.
+        As a result, this function only accepts one to three such
+        integer parameters.
+        """
+
+        args_len = len(args)
+        if args_len == 0:
+            raise ValueError(
+                "please provide at least one integer for `Labels.arange()`"
+            )
+        if args_len > 3:
+            raise ValueError(
+                f"the maximum number of integer arguments accepted by \
+                `Labels.arange()` is 3. {args_len} were provided"
+            )
+        for arg in args:
+            if not np.issubdtype(type(arg), np.integer):
+                raise ValueError(
+                    "all numbers provided to `Labels.arange()` must be integers"
+                )
+
+        labels = Labels(
+            names=[name], values=np.arange(*args, dtype=np.int32).reshape(-1, 1)
```

Comment:
Did we remove the need to specify dtype here @Luthaf ?

Reply:
Maybe use `np.array(range(*args))[:, np.newaxis]`, then you are sure that everything is an integer.

Reply:
I've tried to remove the np.int32 there and everywhere else in equistore